benchmarking/parameter.py

```diff
@@ -0,0 +1,66 @@
+class Parameter:
```

Comment:
Consider using configparser instead of using a python file as config.

Reply:
Done